### DISCUSSION TITLE
I can't center the column header

### DISCUSSION THREAD

**Discussion by smmodules** [Power-Components/livewire-powergrid#1588] [Q&A]
Hey, guys

Can you help me with this? I can't center the column headers.

Likewise, I can't center the toggle button.

`
            Column::make('Peso', 'weight')
                ->withSum('Soma dos Pesos', header: true, footer: false)
                ->bodyAttribute('!text-center')
                ->headerAttribute('!text-end')
                ->editOnClick(
                    hasPermission: true,fallback: 0,saveOnMouseOut: true
                ),
`

`
            Column::add()
                ->headerAttribute('!text-center')
                ->title('Habilitado?')
                ->field('enabled')
                ->toggleable(
                    hasPermission: true,
                    trueLabel: 'Yes',
                    falseLabel: 'No'
                ),
`

![2024-06-12](https://github.com/Power-Components/livewire-powergrid/assets/84677076/513f8e1e-fd98-4bcd-ac14-5b8490b75c35)

**Comment by alvaroarizadadisa** [comment]
Did you solve it? I need it too

**Reply by zikrimuhammad09** [reply]
Up @luanfreitasdev

**Comment by eafarooqi** [comment]
This is simply working for me on bootstrap for toggle button. The problem can be only with tailwind.

```
Column::add()
	->title('WS OK')
	->field('planned')
	->headerAttribute('text-center')
	->bodyAttribute('text-center')
	->toggleable(),
```

![image](https://github.com/user-attachments/assets/1eb09fe4-779b-4e96-9d1d-a08875ef6ffc)

**Comment by spiagz** [comment]
**I was able to fix it by modifying the `cols()` method in my Custom PowerGrid Tailwind Theme.** 

The issue was that the `<tr>` contained a child `<div>` with `flex` class, which was preventing the `text-center` alignment form working due to the `flex` display.

![image](https://github.com/user-attachments/assets/0ec41358-2445-411c-8120-288b6f7236df)

Adding `justify-center` to the class definition solved it.

Here’s the updated method in the custom theme:

```php 
  public function cols(): array
    {
        return [
            'div' => 'select-none flex items-center justify-center gap-1',
        ];
    }

```

This ensures that the elements inside the `<div>` are properly centered. 

Here's a screenshot of how it looks now:
![image](https://github.com/user-attachments/assets/18617488-2667-4fd0-bb84-00f24e0134ac)

Hope this helps!
